### DISCUSSION TITLE
Histograms

### DISCUSSION THREAD

**Discussion by ptman** [plausible/analytics#1344] [Self-Hosted Support] [answered]
How would one view (for a single-market and single-timezone page) which times during weekdays are more active on average?

**Comment by metmarkosaric** [comment, marked as answer]
this feature is not available in Plausible yet. the data is there in the daily charts and you can manually discover some trends but we don't have a nice visualization that makes it easier for you to find the insights. check this thread for details on this feature request https://github.com/plausible/analytics/discussions/754
